app/dashboard/views.py

```diff
@@ -542,43 +546,55 @@ def save_search(request):
 
 @require_POST
 @csrf_exempt
-@ratelimit(key='ip', rate='2/s', method=ratelimit.UNSAFE, block=True)
+@ratelimit(key='ip', rate='5/s', method=ratelimit.UNSAFE, block=True)
 def sync_web3(request):
     """ Sync up web3 with the database.  This function has a few different uses.  It is typically
         called from the front end using the javascript `sync_web3` function.  The `issueURL` is
         passed in first, followed optionally by a `bountydetails` argument.
     """
     # setup
-    result = {}
-    issueURL = request.POST.get('issueURL', False)
-    bountydetails = json.loads(request.POST.get('bountydetails', "{}"))
-
-    if issueURL:
-        issueURL = normalizeURL(issueURL)
-
-        if not bountydetails:
-            # create a bounty sync request
-            result['status'] = 'OK'
-            for existing_bsr in BountySyncRequest.objects.filter(github_url=issueURL, processed=False):
-                existing_bsr.processed = True
-                existing_bsr.save()
+    result = {
+        'status': '400',
+        'msg': "bad request"
+    }
+
+    issueURL = request.POST.get('url', None)
```

Comment:
You can use `issueURL = request.POST.get('url')` and it'll default to `None` without specifying.